site/en/docs/devtools/recorder/reference/index.md

```diff
@@ -63,33 +64,50 @@ You can export and import user flows in the Recorder. This is useful for bug rep
-    - **Export as a Puppeteer script**. Download the recording as a [Puppeteer](https://pptr.dev/) script.
+1. Click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/4dU9UXvsinS4zbgjd8rK.svg", alt="File download.", width="20", height="20" %} **Export** at the top of the **Recorder** panel.
+    {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/wLQZMa2kSWySUn4kdU0Y.png", alt="Export format options.", width="800", height="662" %}
+1. Select one of the following formats from the drop-down list:
+    - **JSON file**. Download the recording as a JSON file.
+    - **@puppeteer/replay**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script.
+    - **Puppeteer**. Download the recording as a [Puppeteer](/docs/puppeteer/) script.
+    - **Puppeteer (including Lighthouse analysis)**. Download the recording as a [Puppeteer](/docs/puppeteer/) script with an embedded [Lighthouse](/docs/lighthouse/) analysis.
+    - One or more options provided by [Recorder Extensions](//docs/devtools/recorder/extensions/#export-extensions).
 1. Save the file.
 
-You can do the following with each export option:
+You can do the following with each default export option:
 
-- **JSON:** Edit the human-readable JSON object and [import](#import-flows) the JSON file back to the **Recorder**.
-- **@puppeteer/replay:** Replay the script with the [Puppeteer Replay](https://github.com/puppeteer/replay) library. When exporting as a @puppeteer/replay script, the steps remain a JSON object. This option is perfect if you want to integrate with your CI/CD pipeline but still have the flexibility to edit the steps as JSON, later convert and import them back into the **Recorder**.
-- **Puppeteer script:** Replay the script with [Puppeteer](https://pptr.dev/). Since the steps are converted into JavaScript, you can have more fine-grained customization, for example, looping the steps. One caveat, you can't import this script back into the **Recorder**.
+- **JSON**. Edit the human-readable JSON object and [import](#import-flows) the JSON file back to the **Recorder**.
+- **@puppeteer/replay**. Replay the script with the [Puppeteer Replay](https://github.com/puppeteer/replay) library. When exporting as a @puppeteer/replay script, the steps remain a JSON object. This option is perfect if you want to integrate with your CI/CD pipeline but still have the flexibility to edit the steps as JSON, later convert and import them back into the **Recorder**.
+- **Puppeteer script**. Replay the script with [Puppeteer](/docs/puppeteer/). Because the steps are converted into JavaScript, you can have more fine-grained customization, for example, looping the steps. One caveat, you can't import this script back into the **Recorder**.
+- **Puppeteer (including Lighthouse analysis)**. This export option is the same as the previous one but it includes a [Lighthouse](/docs/lighthouse/) analysis.
```

Comment:
```suggestion
- **Puppeteer (including Lighthouse analysis)**. This export option is the same as the previous one but it includes a [Lighthouse](/docs/lighthouse/) analysis.
```
Is "includes a Lighthouse analysis" the correct term here? It doesn't include an analysis report, but code to excute and generate one.